src/components/Tooltip/Tooltip.tsx

```diff
@@ -539,7 +561,10 @@ const Tooltip = ({
         }
       })
       if (newAnchors.length) {
```

Comment:
You need to either remove this `if` statement or make it
```suggestion
      if (newAnchors.length || removedAnchors.length) {
```
otherwise it would still cause memory leaks, since if nothing was added and something has been removed, it would just do nothing and the removed elements would be hanging there till restart.